test/framework/github.py

```diff
@@ -566,6 +566,38 @@ def run_check(expected_result=False):
         expected_warning = ''
         self.assertEqual(run_check(True), '')
 
+    def test_det_labels(self):
+        """Test for det_labels function."""
+
+        file_info = {'new_folder': [False], 'new_file_in_existing_folder': [True]}
+        self.mock_stdout(True)
+        res = gh.det_labels(file_info, GITHUB_EASYCONFIGS_REPO)
+        self.mock_stdout(False)
```

Comment:
No need for `mock_stdout` at all, since `det_pr_labels` doesn't produce any output at all?